src/main/java/umaikaze/duke/Ui.java

```diff
@@ -0,0 +1,54 @@
+package umaikaze.duke;
+
+import umaikaze.duke.task.Task;
+
+import java.io.PrintStream;
+import java.io.UnsupportedEncodingException;
+import java.util.List;
+
+public class Ui {
```

Comment:
As mentioned in the other classes, `ui.class` can be made significant in handling all user interaction including showing messages. So you spare your efforts in hardcode all the messages in other methods.